### PR TITLE
Alma job names handling

### PR DESCRIPTION
**Pull request by adamshire123**
#### What does this PR do?
Supports storing job names in SSM and including the AWS environment as part of the job name

#### Includes new or updated dependencies?

YES

#### What are the relevant tickets?
https://mitlibraries.atlassian.net/browse/ENSY-178

#### Developer

- [ ] All new ENV is documented in README
- [ ] Stakeholder approval has been confirmed (or is not needed)

#### Code Reviewer

- [ ] The commit message is clear and follows our guidelines (not just this pull request message)
- [ ] There are appropriate tests covering any new functionality
- [ ] The documentation has been updated or is unnecessary
- [ ] The changes have been verified
- [ ] New dependencies are appropriate or there were no changes

### CODE REVIEW

File: lambdas/webhook.py [hunk 1]
```diff
@@ -29,6 +29,10 @@
     logger.info("No Sentry DSN found, exceptions will not be sent to Sentry")
 
 
+class JobTypeError(Exception):
```
Comment (ehanson8) [inline lambdas/webhook.py:32]: Typos for `unknown` and `received`

File: tests/conftest.py [hunk 2]
```diff
@@ -83,6 +85,7 @@ def post_request_callback(request, context):
         "requestContext": {"http": {"method": request.method}},
         "body": request.body.decode(),
     }
+    reload(lambdas.webhook)
```
Comment (ehanson8) [inline tests/conftest.py:88]: Good catch on this, I need to do something similar with one of the `wiley-deposits` fixtures

**Review by ehanson8 — CHANGES_REQUESTED** [review]
Looks good to me, I like the log message updates. Just 2 typos

**Review by ghukill — APPROVED** [review]
Looks good to me.

**Pull request merged**